### PR TITLE
clean up generic apiserver options

### PR DESCRIPTION
**Pull request by deads2k**
Clean up generic apiserver options before we tag any levels.  This makes them more in-line with "normal" api servers running on the platform.

Also remove dead example code.

@sttts

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/server/options/serving.go [hunk 1]
```diff
@@ -73,11 +73,11 @@ func NewSecureServingOptions() *SecureServingOptions {
 	return &SecureServingOptions{
 		ServingOptions: ServingOptions{
 			BindAddress: net.ParseIP("0.0.0.0"),
-			BindPort:    6443,
+			BindPort:    443,
 		},
 		ServerCert: GeneratableKeyCert{
 			PairName:      "apiserver",
-			CertDirectory: "/var/run/kubernetes",
+			CertDirectory: "apiserver.local.config/certificates",
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/pkg/server/options/serving.go]: strange directory name. But this has to be overwritten anyway to something sensible

**Comment by sttts** [comment]
lgtm

**Comment by deads2k** [comment]
@k8s-bot gci gce e2e test this

**Pull request merged**